sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/http/HttpTimeoutPolicyDefault.java

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.implementation.http;
+
+import io.netty.handler.codec.http.HttpMethod;
+
+
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class HttpTimeoutPolicyDefault extends HttpTimeoutPolicy {
+
+    public static final HttpTimeoutPolicy INSTANCE = new HttpTimeoutPolicyDefault();
+
+    private HttpTimeoutPolicyDefault() {
+    }
+
+    @Override
+    public List<ResponseTimeoutAndDelays> getTimeoutList() {
+        return Collections.unmodifiableList(Arrays.asList(new ResponseTimeoutAndDelays(Duration.ofSeconds(65), 0),
```

Comment:
I thought I saw some discussion around 60s(Java SDK default) or 65s, and we are going to keep use 60s?